proxy/Dockerfile

```diff
@@ -1,7 +1,12 @@
-FROM python:3.10-alpine
+FROM python:3.13-alpine
 WORKDIR /app
-COPY requirements.txt /app/requirements.txt
+COPY ./proxy/requirements.txt /app/requirements.txt
```

Comment:
I assume most would build the container from the directory that contains the Dockerfile (proxy), which is where my automation goes to build and push the proxy container. This would break that. Did you have another idea?

Reply:
I agree we should leave this alone. I did this because I needed to test changes to the proxy AND pypowerwall itself and I didn't know of another good way to build them together, did I miss the memo on how to do that?

Anyway, I think what I should do is move this configuration to a different Dockerfile and leave this one alone.

Reply:
Fair point. My dev and beta build script copies the library before running the build (`cp -r ../pypowerwall .`)